util/src/main/java/tc/oc/pgm/util/material/ColorUtils.java

```diff
@@ -29,6 +29,8 @@ public interface ColorUtils {
 
   boolean isColor(BlockState block, DyeColor color);
 
+  boolean isColor(ItemStack stack, DyeColor color);
```

Comment:
This should probably just use pgm's MaterialData as param instead, so it can work for both blockstte and itemstacks: `MaterialData.of(blockstate)` or `MaterialData.of(stack)`